### ISSUE TITLE
Same device included 'indefintely'

### ISSUE THREAD

**Issue by sebcaps** [zwave-js/zwavejs2mqtt#2111]
Build/Run method

- [X ] Docker
- [ ] PKG
- [ ] Snap package
- [ ] Manually built (git clone - yarn install - yarn run build )

Zwavejs2Mqtt version: 6.2.0
Z-Wave JS version: 8.9.1
### Describe the bug

I included a new device (Fibaro FGR-223) using QRCode text and my device get inclued 'indefinitely' 
![image](https://user-images.githubusercontent.com/24374878/147495070-ff1f2e7d-6d67-4d18-8cc4-6697925dd47a.png)

### To Reproduce

1. Go to Node manager
2. Click on inclusion
3 Select QR Code 
4 Click text
5. input text
6. Validate

### Expected behavior

Just one device added

**Comment by robertsLando** [comment]
@sebcaps does a refresh of the page fix the issue? Could you add a log? Never seen something like that before

**Comment by sebcaps** [comment]
Hello, heres some logs.
Since I choose verbose level; I took some quick snapshot of logs et merge them in a single file.
hope it'll help.
In verbose level web site is unresponsive...
Try to go to a lower level.
[Logs.txt](https://github.com/zwave-js/zwavejs2mqtt/files/7783785/Logs.txt)

I assume not able to include my device using the B button, for the moment only QR code seems to get the device

**Comment by robertsLando** [comment]
@sebcaps Logs seems truncated, please follow guide on docs to export them: https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/generating-logs?id=driver-logs

We need DRIVER LOGS

**Comment by sebcaps** [comment]
Indeed... Here is the complete file.
Set log level to silly, but file content does not seems like logs output on window. Well let's see if you find any useful data :-).
Sorry for the wrong first file
[zwavejs2mqtt-store.zip](https://github.com/zwave-js/zwavejs2mqtt/files/7783924/zwavejs2mqtt-store.zip)
.

**Comment by sebcaps** [comment]
Here are the files.
I missed the second one at first try...
Sorry for the inconvenience.
Hope it will be usefu
[zwavejs_2021-12-28.zip](https://github.com/zwave-js/zwavejs2mqtt/files/7783979/zwavejs_2021-12-28.zip)
[zwavejs2mqtt_2021-12-28.zip](https://github.com/zwave-js/zwavejs2mqtt/files/7783980/zwavejs2mqtt_2021-12-28.zip)
l

**Comment by robertsLando** [comment]
@sebcaps did you use default inclusion or replace failed node?

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by sebcaps** [comment]
Inclusion using DSK code. (not used replace failed node)

**Comment by AlCalzone** [comment]
Wow, that is an interesting one. Whenever the driver tries to query the state of the `Protection CC`, the device seems to crash and factory reset. Then it sends another SmartStart inclusion request, it gets included, the driver interviews it until it gets to the Protection CC and the cycle starts anew.

This is something Fibaro need to fix (or maybe your device is faulty). I'm almost sure they're going to try and claim its not their fault though. This log snippet highlights it nicely, I suggest sending it to them along with the support inquiry:
```
2021-12-28T09:33:14.955Z DRIVER « [REQ] [ApplicationUpdateRequest]
                                    type:                  SmartStart_HomeId_Received
                                    remote node ID:        0
                                    NWI home ID:           0xfd1d4842
                                    basic device class:    4
                                    generic device class:  17
                                    specific device class: 6
                                    supported CCs:
                                    · Z-Wave Plus Info
                                    · Multilevel Switch
                                    · Association
                                    · Multi Channel Association
                                    · Association Group Information
                                    · Transport Service
                                    · Version
                                    · Manufacturer Specific
                                    · Device Reset Locally
                                    · Powerlevel
                                    · Security
                                    · Security 2
                                    · Meter
                                    · Configuration
                                    · CRC-16 Encapsulation
                                    · Notification
                                    · Protection
                                    · Multi Channel
                                    · Central Scene
                                    · Firmware Update Meta Data
                                    · Application Status
                                    · Supervision
2021-12-28T09:33:14.961Z CNTRLR   Received Smart Start inclusion request
2021-12-28T09:33:14.963Z CNTRLR   NWI Home ID found in provisioning list, including node...


... device gets included and interviewed - takes some time...


2021-12-28T09:33:54.570Z CNTRLR   [Node 084] Interviewing Protection...
2021-12-28T09:33:54.570Z CNTRLR » [Node 084] querying protection capabilities...
2021-12-28T09:33:54.592Z SERIAL » 0x01090013540275042513f4                                            (11 bytes)
2021-12-28T09:33:54.593Z DRIVER » [Node 084] [REQ] [SendData]
                                  │ transmit options: 0x25
                                  │ callback id:      19
                                  └─[ProtectionCCSupportedGet]
2021-12-28T09:33:54.600Z SERIAL « [ACK]                                                                   (0x06)
2021-12-28T09:33:54.616Z SERIAL « 0x0104011301e8                                                       (6 bytes)
2021-12-28T09:33:54.617Z SERIAL » [ACK]                                                                   (0x06)
2021-12-28T09:33:54.620Z DRIVER « [RES] [SendData]
                                    was sent: true
2021-12-28T09:33:54.632Z SERIAL « 0x011800131300000300c77f7f7f7f010103000000000201000023              (26 bytes)
2021-12-28T09:33:54.632Z SERIAL » [ACK]                                                                   (0x06)
2021-12-28T09:33:54.637Z DRIVER « [REQ] [SendData]
                                    callback id:            19
                                    transmit status:        OK, took 30 ms
                                    routing attempts:       1
                                    protocol & route speed: Z-Wave, 40 kbit/s
                                    ACK RSSI:               -57 dBm
                                    ACK channel no.:        1
                                    TX channel no.:         1
2021-12-28T09:33:54.659Z SERIAL « 0x010f000400540775050005000300c70016                                (17 bytes)
2021-12-28T09:33:54.675Z SERIAL » [ACK]                                                                   (0x06)
2021-12-28T09:33:54.696Z DRIVER « [Node 084] [REQ] [ApplicationCommand]
                                  └─[ProtectionCCSupportedReport]
                                      supports exclusive control: false
                                      supports timeout:           false
                                      local protection states:
                                      · Unprotected
                                      · NoOperationPossible
                                      RF protection states:
                                      · Unprotected
                                      · NoControl
2021-12-28T09:33:54.718Z CNTRLR « [Node 084] received protection capabilities:
                                  exclusive control:       false
                                  timeout:                 false
                                  local protection states: 
                                  · Unprotected
                                  · NoOperationPossible
                                  RF protection states:    
                                  · Unprotected
                                  · NoControl
2021-12-28T09:33:54.719Z CNTRLR » [Node 084] querying protection status...
2021-12-28T09:33:54.736Z SERIAL » 0x01090013540275022514f5                                            (11 bytes)
2021-12-28T09:33:54.736Z DRIVER » [Node 084] [REQ] [SendData]
                                  │ transmit options: 0x25
                                  │ callback id:      20
                                  └─[ProtectionCCGet]
2021-12-28T09:33:54.780Z SERIAL « [ACK]                                                                   (0x06)
2021-12-28T09:33:54.782Z SERIAL « 0x0104011301e8                                                       (6 bytes)
2021-12-28T09:33:54.782Z SERIAL » [ACK]                                                                   (0x06)
2021-12-28T09:33:54.783Z DRIVER « [RES] [SendData]
                                    was sent: true


... short pause here, device crashes and resets...


2021-12-28T09:34:01.223Z SERIAL « 0x0124004985001efd1d4842160411065e26858e595586725a73989f32705671756 (38 bytes)
                                  05b7a226c5c
2021-12-28T09:34:01.225Z SERIAL » [ACK]                                                                   (0x06)
2021-12-28T09:34:01.227Z DRIVER « [REQ] [ApplicationUpdateRequest]
                                    type:                  SmartStart_HomeId_Received
                                    remote node ID:        0
                                    NWI home ID:           0xfd1d4842
                                    basic device class:    4
                                    generic device class:  17
                                    specific device class: 6
                                    supported CCs:
                                    · Z-Wave Plus Info
                                    · Multilevel Switch
                                    · Association
                                    · Multi Channel Association
                                    · Association Group Information
                                    · Transport Service
                                    · Version
                                    · Manufacturer Specific
                                    · Device Reset Locally
                                    · Powerlevel
                                    · Security
                                    · Security 2
                                    · Meter
                                    · Configuration
                                    · CRC-16 Encapsulation
                                    · Notification
                                    · Protection
                                    · Multi Channel
                                    · Central Scene
                                    · Firmware Update Meta Data
                                    · Application Status
                                    · Supervision


The previous query fails:
2021-12-28T09:34:03.313Z SERIAL « 0x0118001314010358007f7f7f7f7f01010700000000020a0000ca              (26 bytes)
2021-12-28T09:34:03.315Z SERIAL » [ACK]                                                                   (0x06)
2021-12-28T09:34:03.317Z DRIVER « [REQ] [SendData]
                                    callback id:            20
                                    transmit status:        NoAck, took 8560 ms
                                    routing attempts:       10
                                    protocol & route speed: Z-Wave, 40 kbit/s
                                    TX channel no.:         1
```

**Comment by robertsLando** [comment]
Wow that's awesome 😆

**Comment by sebcaps** [comment]
OK thx for your help.
I'll give a try to fibaro and let you know.
Could it be caused by a wrong device wiring ?

**Comment by AlCalzone** [comment]
I don't think so. It works perfectly fine until that one command is received.

**Comment by sebcaps** [comment]
Hello, FYI, I tried to re-appair another device using QR Code input (device was OK but lost due to multiple tries) and guess what : same issue. Two faulty devices looks very strange, I think I did the same wrong actions,  which ?
 I don't know exactly ;-) . this fu*** B-button is not quite accessible.
In any case my both devices are off and I'll need to give a try to 'basic' inclusion.

**Comment by AlCalzone** [comment]
I wonder if that changes things. I can't see anything you did wrong.

**Comment by sebcaps** [comment]
Hi, I did a "new try" cause I hate when it does not work.
But here I set security keys (settings -> S2 unauthenticated, S2 authenticated, S2 access control and S0 legacy). Previously theses values were not set => in that case everything ran smoothly - still using DSK mode-  and my 2 devices appaired and working correctly. I don't know if there"s a link but definitely the only change before it worked correctly. 
My bad I did not enable logging to allow deeper investigation/explanation
Many thx for your support and availbility.

Off topic & personal consideration : I have zwave & zigbee devices, to my experience zigbee init process is very, very, very more easy.
